dali/pipeline/operators/decoder/cache/decoder_cache_blob_test.cc

```diff
@@ -65,22 +60,20 @@ TEST_F(DecoderCacheBlobTest, AddExistingIgnored) {
 }
 
 TEST_F(DecoderCacheBlobTest, ErrorTooSmallCacheSize) {
-  SetUpImpl(kValue1.size()-1);
+  SetUpImpl(kValue1.size() - 1);
   cache_->Add(kKey1, &kValue1[0], kValue1.size(), kDims1);
   EXPECT_FALSE(cache_->IsCached(kKey1));
 }
 
 TEST_F(DecoderCacheBlobTest, AllocateMoreThan2000MB) {
```

Comment:
Just to make sure: `static_assert(sizeof(size_t) > 4)`

Reply:
in here or somewhere else?